src/idl_parser.cpp

```diff
@@ -3064,9 +3064,29 @@ CheckedError Parser::DoParse(const char *source, const char **include_paths,
       EXPECT(';');
     } else if (IsIdent("include")) {
       return Error("includes must come before declarations");
-    } else if (IsIdent("attribute")) {
+    }  //
+    else if (IsIdent("attribute")) {
       NEXT();
       auto name = attribute_;
+      // support go module define in attribute
+      // attribute "go_module:github.com/google/flatbuffers/go-example/";
```

Comment:
The `attribute` keyword is used to declare attributes, you can't just reuse it for go modules.
instead, make a new top level statement like `go_module "github.com/google/flatbuffers/go-example/"`.
That way you also don't require your split.
Either way, can all go module changes be moved into their own PR, so this all becomes easier to review?

Reply:
a new top level statement like go_module ...... it's good for me.